authors/apps/authentication/tests/test_email_verification.py

```diff
@@ -0,0 +1,17 @@
+from django.urls import reverse
+from rest_framework import status
+
+# local import
+from .test_config import TestConfiguration
+
+
+class TestEmailVerification(TestConfiguration):
+    """
+    Tests for email verification
+    """
+
+    def test_email_verification(self):
+        """
+        Test email verification link is sent to the user
+        """
+        pass
```

Comment:
@Darius-Ndubi, this test doesn't seem implemented. What's being tested here?

Reply:
@evansmusomi, let me fix that.